dask/dataframe/tests/test_dataframe.py

```diff
@@ -2644,3 +2644,17 @@ def test_del():
 
     del df['x']
     assert_eq(a, df)
+
+
+@pytest.mark.parametrize('index', [True, False])
+@pytest.mark.parametrize('deep', [True, False])
+def test_memory_usage(index, deep):
+    df = pd.DataFrame({'x': [1, 2, 3],
+                       'y': [1.0, 2.0, 3.0],
+                       'z': ['a', 'b', 'c']})
+    ddf = dd.from_pandas(df, npartitions=2)
+
+    assert_eq(df.memory_usage(index=index, deep=deep),
+              ddf.memory_usage(index=index, deep=deep))
```

Comment:
We normally don't use compute with assert_eq, it wants the full dataframe object so that it can run  a variety of tests on it.  This is true both for arrays and dataframe (each of which have different assert_eq functions).  In the series case the result is a delayed object.  Actually, I should probably just use normal `assert` there